src/Components/Nav/LoginModal.js

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import './LoginModal.scss';
+import 'Styles/common.scss';
+
+const LoginModal = () => {
+  const [inputValue, setInputValue] = useState({
+    email: '',
+    password: '',
+  });
+
+  const { email, password } = inputValue;
+  const isInputValid = email.includes('@') && password.length >= 5;
+
+  function handleInputValue(e) {
+    const { name, value } = e.target;
+    setInputValue(inputValue => ({ ...inputValue, [name]: value }));
```

Comment:
함수형 업데이트를 사용하지 않아도 될 것 같네요!
```suggestion
    setInputValue(inputValue => ({ ...inputValue, [name]: value }));
```
```suggestion
    setInputValue({ ...inputValue, [name]: value });
```

Reply:
네 반영했습니다!